src/components/Switch/switch.component.css

```diff
@@ -0,0 +1,51 @@
+.inputWrapper {
+    position: relative;
+    top: 5px;
+    left: 5px;

```

Comment:
Our preference is to use `rem` rather than `px`. We like to change the `rem` baseline, so that `1rem` = `10px` with the following CSS:

```
html {
  font-size: 62.5%;
}
```

Is this something you would like to do in this PR? Or shall we leave it for a future practice story?

Reply:
PR means Peer Review? I have just changed the CSS file to make rem easier to scale.

Reply:
PR means "Pull Request" in this context.